egs/sprakbanken_swe/README.txt

```diff
@@ -0,0 +1,10 @@
+About the sprakbanken corpus:
+    This corpus is a free corpus originally collected by NST for ASR purposes and currently 
+    hosted by the Norwegian libraries. The corpus is multilingual and contains Swedish, 
+    Norwegian (Bokmål) and Danish. This setup uses the Swedish subcorpus. The vocabulary is 
+    large and there is approx. 480 hours of read-aloud speech with associated text scripts.
+
+    Some months ago the corpus was republished here: http://www.nb.no/sprakbanken/#ticketsfrom?lang=en
+
+    s5: This is the current recommended recipe. (Swedish) 
+
```

Comment:
Peerhaps you could mention the recipe was created by you from the other sprakbank?